packages/dev/test/mock-api.ts

```diff
@@ -0,0 +1,136 @@
+import type { IncomingHttpHeaders } from 'http'
```

Comment:
This file has been copied wholesale from the CLI. I just replaced the native HTTP server from Node.js with the `HTTPServer` class from `@netlify/dev-utils`.